src/content/icons/dropdawn-chevron-down.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Did you mean "dropdown" ? It is not the Dawn of the Dead as far as I know 😉

Reply:
Not me, that was Alain who gave names for those files I've just imported them from zeroheight. I will correct